witnet.toml

```diff
@@ -26,7 +26,7 @@ update_period_seconds = 8000000
 
 [mining]
 enabled = true
-data_request_timeout_seconds = 5
+data_request_timeout_milliseconds = 1000
```

Comment:
1 second timeout may be too aggressive, thoughts @lrubiorod?

Reply:
I was using 3 seconds in my branch :P

Reply:
What about `1500`? I'm ready to haggle every single millisecond!

Reply:
Some input:
- At `1000`, the BTC price example resolves to `Timeout` in around ~3% of the attempts.
- At `2000`, the sample example resolves to `Timeout` in a solid 0% of the attempts.